src/cmr/concepts/acl.js

```diff
@@ -27,48 +31,171 @@ export default class Acl extends Concept {
     return items
   }
 
-  fetch(searchParams) {
-    const params = mergeParams(searchParams)
-
-    // Default an array to hold the promises we need to make depending on the requested fields
-    const promises = []
-
-    const { jsonKeys } = this.requestInfo
-
-    if (jsonKeys.length > 0) {
-      const jsonHeaders = {
-        ...this.headers
-      }
-      promises.push(
-        this.fetchAcl(params, jsonKeys, jsonHeaders)
-      )
-    } else {
-      // Push a null promise to the array so that the umm promise always exists as
-      // the second element of the promise array
-      promises.push(
-        // eslint-disable-next-line no-promise-executor-return
-        new Promise((resolve) => resolve(null))
-      )
-    }
-
-    this.response = Promise.all(promises)
-  }
-
   /**
    * Query the CMR JSON API endpoint to retrieve requested data
    * @param {Object} searchParams Parameters provided by the query
    * @param {Array} requestedKeys Keys requested by the query
    * @param {Object} providedHeaders Headers requested by the query
    */
-  fetchAcl(searchParams, requestedKeys, providedHeaders) {
+  fetchJson(searchParams, requestedKeys, providedHeaders) {
     this.logKeyRequest(requestedKeys, 'json')
 
-    // Construct the promise that will request data from the json endpoint
-    return aclQuery({
-      conceptType: this.getConceptType(),
-      params: pick(snakeCaseKeys(searchParams), this.getPermittedJsonSearchParams()),
+    const defaultSearchParams = {
+      include_full_acl: true
+    }
+
+    const { conceptId } = searchParams
+
+    if (conceptId) {
+      // eslint-disable-next-line no-param-reassign
+      searchParams.id = conceptId
+
+      // eslint-disable-next-line no-param-reassign
+      delete searchParams.conceptId
```

Comment:
Should we create a new object that only has what we need, rather than manipulating one with a `delete` to remove the things we don't?